providers/shared/attributesets/event/id.ftl

```diff
@@ -0,0 +1,38 @@
+[#ftl]
+
+[@addAttributeSet
+    type=DB_SYSTEM_EVENT_ATTRIBUTESET_TYPE
+    properties=[
+        {
+                "Type"  : "Description",
+                "Value" : "Configuration of reported system notifications from component resources"
+        }]
+    attributes=[
+        {
+            "Names" : "Categories",
+            "Description" : "Optional array of event categories to report",
+            "Types" : ARRAY_OF_STRING_TYPE,
+            "Values" : [
+                "availability", 
+                "backup", 
+                "configuration change", 
+                "creation", 
+                "deletion", 
+                "failover", 
+                "failure", 
+                "low storage", 
+                "maintenance", 
+                "notification", 
+                "read replica", 
+                "recovery", 
+                "restoration", 
+                "security"
+            ]
+        },
+        {
```

Comment:
Is this optional? wouldn't default be all of them?

Reply:
Yes on both counts

Reply:
So should we just make the default all of them? 
To me an undefined value would mean none not all.

Reply:
The AWS processing is an absent Categories attribute means all events (presumably to allow for them adding new ones) ie it's a filtering attribute. If we default the value, we lose the "all events even ones not yet defined" status. To have no events processed, you'd need to remove the SystemNotification attribute level. Thoughts?

Reply:
Maybe an _all option as the default. Then when you pass that to AWS it is sent as the empty string. 
Probably also need a check that someone hasn't set _all and other values.

Reply:
For that to make sense, we'd need to make Categories mandatory, correct? So the only way to get no Categories attribute is to specify the "_all" value?

Reply:
Include _all in the values ad add a Default property on the account. 
That would mean that its always _all by default and then you can override that with specific categories as required.

Reply:
Or just to have left it as an empty attribute not passed through, but done

Reply:
Then it wouldn't show in documentation. Trying to get the understanding of what it does without making people read the code.